### PR TITLE
Add parsing

### PR DESCRIPTION
**Pull request by in-siders17**

### CODE REVIEW

File: message.py [hunk 1]
```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
...
-
+    parser = argparse.ArgumentParser(description='Sending email')
+    parser.add_argument('-s', dest="sender", help='sender')
+    parser.add_argument('-p', dest="password", help='password')
+    parser.add_argument('-r', dest="receiver", help='receiver')
+    parser.add_argument('-m', dest="message", help='message')
+    args = parser.parse_args()
+    send_email(sender=args.sender, password=args.password, receiver=args.receiver, message=args.message)
 
 if __name__ == "__main__":
```
Comment (tenk28) [inline message.py:29]: test comment

**Review by tenk28 — COMMENTED** [review]
Test commit

File: message.py [hunk 2]
```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
...
-    message = input("Type your message: ")
-    print(send_email(sender, password, receiver, message))
-
+    parser = argparse.ArgumentParser(description='Sending email')
+    parser.add_argument('-s', dest="sender", help='sender')
+    parser.add_argument('-p', dest="password", help='password')
+    parser.add_argument('-r', dest="receiver", help='receiver')
+    parser.add_argument('-m', dest="message", help='message')
+    args = parser.parse_args()
+    send_email(sender=args.sender, password=args.password, receiver=args.receiver, message=args.message)
```
Comment (tenk28) [inline message.py]: Too long row, you can remove sender=*, ...
Better rewrite to _send_email(args.sender, args.password, args.receiver, args.message)_

File: message.py [hunk 3]
```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
...
-
 def main():
-    sender = "boyhich3@gmail.com"
-    password = "tmtzabdqcjmmrpwu"
-    receiver = "shtrikker28@gmail.com"
-    message = input("Type your message: ")
-    print(send_email(sender, password, receiver, message))
-
+    parser = argparse.ArgumentParser(description='Sending email')
+    parser.add_argument('-s', dest="sender", help='sender')
```
Comment (tenk28) [inline message.py]: help parameter must be more detailed and short: such as 
"Email, who send"

File: message.py [hunk 4]
```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
...
+    parser.add_argument('-p', dest="password", help='password')
```
Comment (tenk28) [inline message.py]: Same with password, receiver and message

File: message.py [hunk 5]
```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
         server.login(sender, password)
         server.sendmail(sender, receiver, message)
 
-        return "The message was sent succesfuly!"
+        return "The message was sent successfully!"
     except Exception as _ex:
```
Comment (dindin28) [inline message.py:13]: Let's rename to _ex_ without _

File: message.py [hunk 6]
```diff
@@ -1,4 +1,5 @@
 import smtplib
+import argparse
 
 def send_email(sender, password, receiver, message):
     server = smtplib.SMTP("smtp.gmail.com", 587)
```
Comment (tenk28) [inline message.py]: Create one more file, name it "config.py", put there hardcoded variables "smtp.gmail.com" and 587

File: message.py [hunk 7]
```diff
@@ -8,18 +9,18 @@ def send_email(sender, password, receiver, message):
         server.login(sender, password)
         server.sendmail(sender, receiver, message)
 
-        return "The message was sent succesfuly!"
+        return "The message was sent successfully!"
```
Comment (tenk28) [inline message.py]: Let's return "true" if message was sent and "false" if not

**Pull request merged**